frontend21/scss/components/section.scss

```diff
@@ -0,0 +1,44 @@
+.#{prefix(null, 'section')} {
+  width: 100%;
+  margin-top: 100px;
+  padding: 0 15px;
```

Comment:
Is there a need for additional padding? Doesn't the `grid-gap` from the parent grid create enough spacing?